CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   [issue](https://github.com/ComplianceAsCode/compliance-operator/issues/8) for
   more details.
 
+- the upstream catalog image is now built using the
```

Comment:
Just capitalize `The` and the rest is good to me.

Reply:
thanks for the review, done.